### PR TITLE
added HowToPlayComponent and changed the title

### PR DESCRIPTION
**Pull request by skarthikeyan96**
### Guidelines for creating Pull Requests
I have added the howtoplay component and also changed the title in heading. I have attached the screenshot below

screenshot 12

Thanks for contributing to the project. Please follow the guidelines
below:

* Make sure you've read the [Contributing guide](https://github.com/UdacityMobileWebScholarship/guess-quote/blob/master/CONTRIBUTING.md).
* Describe how this PR will make our project even more awesome.
* If this solves an issue, please reference it in this PR.

### CODE REVIEW

**Comment by skarthikeyan96** [comment]
screenshot 15

File: client/src/components/HowToPlayComponent/HowToPlay.css [hunk 1]
```diff
@@ -1,15 +1,18 @@
 .HowToplay {
   font-family: "Segoe UI", Roboto;
   font-size: 1rem;
-
+  background-color: #9b59b6;
```
Comment (twishasaraiya) [inline client/src/components/HowToPlayComponent/HowToPlay.css]: Please use color code `#35009D` for background and make it fill the entire background and not half.

**Comment by skarthikeyan96** [comment]
Sure will change it

File: client/src/components/HowToPlayComponent/HowToPlay.css (hunk 1 above)
Comment (skarthikeyan96) [reply client/src/components/HowToPlayComponent/HowToPlay.css]: Sure will change it

**Comment by skarthikeyan96** [comment]
screenshot 16

File: client/src/containers/SampleContainer/SampleContainer.js [hunk 2]
```diff
@@ -16,6 +17,7 @@ class SampleContainer extends PureComponent {
     return (
       <div className="SampleContainer">
         <SampleComponent message={this.props.sample.message} />
+        <HowToPlay />
```
Comment (skyshader) [inline client/src/containers/SampleContainer/SampleContainer.js]: Let's not use SampleContainer to display HowToPlay Component. This belongs in a separate page and a url.

File: client/src/components/HowToPlayComponent/HowToPlay.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import React from "react";
```
Comment (skyshader) [inline client/src/components/HowToPlayComponent/HowToPlay.js]: Let's just create a page, and link this component there as it is just a static list of information and a page with different route.

File: client/src/containers/SampleContainer/SampleContainer.js (hunk 2 above)
Comment (skarthikeyan96) [reply client/src/containers/SampleContainer/SampleContainer.js]: I will change it and make pr by tonight

**Comment by skarthikeyan96** [comment]
screenshot 17

**Comment by twishasaraiya** [comment]
Looks good to me.@skyshader what do you think??

**Pull request merged**